spring-cloud-kubernetes-fabric8-discovery/src/test/java/org/springframework/cloud/kubernetes/fabric8/discovery/KubernetesDiscoveryClientTest.java

```diff
@@ -16,43 +16,39 @@
 
```

Comment:
This test was failing after I did the above change. The most sane thing to do, imho, is to fix it with the way we have already refactored tests in configmap/secrets implementation. In the long run (once I get here), this entire package should be refactored to junit-5. But in order to fix this current issue, this is OK.

Reply:
It would take a while, but if you look closely, I have not changed the asserts, at all, to preserve the tests logic.